### ISSUE TITLE
Hornet Interface does not export SAI glideslope deviation and pitch adjust wings

### ISSUE THREAD

**Issue by BlueFinBima** [HeliosVirtualCockpit/Helios#681]
If you are having problems installing / configuring / using a profile then you should use the support process for the provider of that profile.

- [x] I have read [Getting Help](https://github.com/HeliosVirtualCockpit/Helios/wiki/Get-Help)

**Describe the Helios Issue**
Standby Attitude Indicator for the hornet does not export the ILS glideslope indicator values or the Pitch Adjust wing value.

**To Reproduce**

**Expected behaviour**
Users should be able to bind the Pitch Adjust wings and ILS glideslope deviations to the Hornet SAI.

**Screenshots**

**please complete the following information:**
 - [x] Profile Editor
 - [x] Control Center

**Additional context**
Raised by Sting#1651 on Discord https://discord.com/channels/434241908492075010/456398012378513417/1056019651961835571

```
Hi all just setting up the standby instruments for the new screen.  Was able get them all working except the SAI.   I have the ball rolling around correctly however I can not find were to map the waterline change and the ils bars etc.  I have been using Helios to complete this.  Is it possible to export a fully functioning SAI?
```

**Comment by BlueFinBima** [comment]
From what I can remember, I deliberately created a minimal SAI for the hornet with the intention of coming back to it at some point.  That point would seem to be now.

**Comment by BlueFinBima** [comment]
Awaiting delivery

**Comment by BlueFinBima** [comment]
Shipped in 1.6.5700.0
